test/functional/eval/glob_spec.lua

```diff
@@ -0,0 +1,20 @@
+local helpers = require('test.functional.helpers')
+local clear, execute, eval, eq = helpers.clear, helpers.execute, helpers.eval, helpers.eq
+
+before_each(function()
+  clear()
+  lfs.mkdir('test-glob')
+  execute('cd test-glob')
+end)
+
+after_each(function()
+  lfs.rmdir('test-glob')
+end)
+
+describe('glob()', function()
+  describe('glob(".*", 0, 1)', function()
```

Comment:
probably don't need the nested describe(). Instead:

    it("glob('.*') returns . and .. " ...

Also, prefer single-quotes for viml string literals in general.